proto.json

```diff
@@ -229,6 +229,12 @@
     "syft.grid.clients.data_centric_fl_client.DataCentricFLClient": {
       "code": 70,
       "forced_code": 71
+    },
+    "syft.grid.clients.model_centric_fl_client.ModelCentricFLClient": {
```

Comment:
In our syft repo we have an explicit ```get_msgpack_code``` method (in the ```grid/clients/ModelCentriFLClient```) which I think is added manually there because we did not have a proto entry for ```ModelCentricFLClient```